### DISCUSSION TITLE
Duda con creacion de make file

### DISCUSSION THREAD

**Discussion by leonelcespedess** [IIC2133-PUC/2024-1#16] [Preguntas materia] [answered]
Para ejecutar el comando make se debe trabajar con un makefile, este lo tenemos que crear de manera manual?  En el video de las cápsulas dice que se debería generar de manera automática el contenido de este, pero no me esta generando nada):

**Comment by CataAlamos** [comment, marked as answer]
Hola! En tu repositorio base debe haber un makefile listo para que lo uses. Nosotros te damos un makefile para compilar tu código, te sugiero evitar modificarlo.

**Reply by leonelcespedess** [reply]
me había olvidado del repo, gracias! Aprovecho d preguntar si es debemos trabajar dentro de src para crear los archivos que necesitemos, ya que vi que ahí estaba el main

**Reply by CataAlamos** [reply]
Hola! Dentro de src hay una carpeta llamada ships, ahi debes hacer tus archivos.
Exito con la tarea!
